### PR TITLE
Register .live domain

### LINKED ISSUE #830: Update add domain to existing site to register (and cancel) .live domain
We can use the sandbox method for adding a fake .live domain to an existing site and purchase it then refund and cancel it.

### PR DESCRIPTION
**Pull request by Stojdza**
Adding test for #830. Existing user should purchase _.live_ domain, and after that cancel and refund it. 

Resolves #830

### CODE REVIEW

File: specs/wp-manage-domains-spec.js [hunk 1]
```diff
@@ -283,4 +291,189 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
 			return await TransferDomainPrecheckPage.Expect( driver );
 		} );
 	} );
+
+	describe.only( 'Register (and cancel) .live domain to an existing site ', function() {
```
Comment (Stojdza) [inline specs/wp-manage-domains-spec.js]: This `.only` is intentional, so we can run only this test with default test user.

File: specs/wp-manage-domains-spec.js [hunk 2]
```diff
@@ -283,4 +291,189 @@ describe( `[${ host }] Managing Domains: (${ screenSize }) @parallel`, function(
...
+				);
+				return this.skip();
+			}
+		} );
+
+		before( async function() {
+			return await driverManager.ensureNotLoggedIn( driver );
+		} );
+
+		step( 'We can visit set the sandbox cookie for payments', async function() {
```
Comment (Stojdza) [inline specs/wp-manage-domains-spec.js]: Sandbox cookie is used like on several different places, but I'm not sure is it set properly to purchase `.live` domain. 
As you can see in the test run purchasing `.live` domain is passing. But when user want to cancel and refund it there is no option for that. 

screen shot 2018-09-04 at 16 20 33

**Comment by alisterscott** [comment]
@Stojdza it looks like it's being set correctly as the sandbox payment is passing. I'm not sure why the cancel option doesn't appear. I can see the domain in S.A as long as I am in sandboxed mode. Maybe as in #domains-dev ? Maybe we don't need to worry about cancelling these?

**Comment by Stojdza** [comment]
@alisterscott I've tried with #domains-dev but they didn't know why is this happening. We wanted here to close the complete loop - adding, canceling and refund. Since these steps are covered separately in other tests and too much time is lost here, maybe it's not urgent and we can leave it for HACK week or meetup?

**Comment by alisterscott** [comment]
> Since these steps are covered separately in other tests and too much time is lost here, maybe it's not urgent and we can leave it for HACK week or meetup?

sounds good, feel free to close it

**Pull request closed without merging**